### ISSUE TITLE
Memory allocation error on import `tensorflow_quantum` on Intel Mac

### ISSUE THREAD

**Issue by greenstick** [tensorflow/quantum#684]
## Summary

As others have noted, I'm unable to install Tensorflow Quantum using Pip (i.e. Pip isn't able to find it). A seeming work-around has been to install `tfq-nightly`, however, when I do, I get a memory allocation error on import. This seems to be a consistent issue as it's persisted over a couple of weeks – it does not seem related to a specific nightly build.

Any ideas on work-arounds or how I could get this resolved would be very helpful.

## Error (and Python + Clang versions)
```
Python 3.9.10 (main, Jan 15 2022, 11:48:00)
[Clang 13.0.0 (clang-1300.0.29.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.

import tensorflow_quantum
Python(6224,0x109877600) malloc: *** error for object 0x1098771c0: pointer being freed was not allocated
Python(6224,0x109877600) malloc: *** set a breakpoint in malloc_error_break to debug
zsh: abort      python3
```
## OS/Device Details
2019 MBP 2.9 GHz Intel Core i9
MacOS Monterey Version 12.3.1

##  Tensorflow Module Versions
tensorboard                               2.8.0
tensorboard-data-server         0.6.1
tensorboard-plugin-wit            1.8.0
tensorflow                                 2.7.0
tensorflow-estimator               2.7.0
tensorflow-io-gcs-filesystem 0.24.0
tensornetwork                          0.4.6
tf-estimator-nightly         2.8.0.dev2021122109
tfq-nightly                  0.6.0.dev20220104

## Attempts Resolve This Issue

Attempt 1:
1. Updated MacOS from Catalina to Monterey (the original issue occurred on MacOS Catalina, was planning on doing this anyway).

Attempt 2:
1. Uninstalling `tfq-nightly` and reinstalling it with the `--no-cache-dir` flag.

Attempt 3:
1. Uninstalling `tfq-nightly`.
2. Updating GCC compiler via Homebrew.
3. Uninstalling and reinstalling Python with updated compilers.
4. Reinstalling `tfq-nightly` with the `--no-cache-dir` flag.

Admittedly, these attempts haven't been particularly methodical...nor have they done anything to resolve the issue.

**Comment by lockwo** [comment]
Using python 3.9 I replicated your error. However, I have always fixed it simply by using python 3.8 instead (just checked again and it worked for me). This is on a 2019 MBP 2.4 GHz intel core i9, Monterey 12.2.1. 

Other relevant Mac discussions: https://github.com/tensorflow/quantum/issues/682#issuecomment-1095096061

**Comment by greenstick** [comment]
Thanks @lockwo, that worked!
